lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -65,8 +71,12 @@ public APIGatewayProxyResponseEvent handleRequest(
                     ipvSessionService.getIpvSession(ipvSessionId).getClientSessionDetails();
             String userId = clientSessionDetailsDto.getUserId();
 
-            LogHelper.attachGovukSigninJourneyIdToLogs(
-                    clientSessionDetailsDto.getGovukSigninJourneyId());
+            String govukSigninJourneyId = clientSessionDetailsDto.getGovukSigninJourneyId();
+            LogHelper.attachGovukSigninJourneyIdToLogs(govukSigninJourneyId);
+
+            List<ContraIndicatorItem> ciItems =
+                    getCIsAndSwallowErrors(userId, govukSigninJourneyId);
+            LOGGER.info("Retrieved {} CI items", ciItems.size());
```

Comment:
I think keep this here for now. I've added a note to the ticket to start using these CIs to remove it.